src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,58 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ onAdd }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [errorMessage, setErrorMessage] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleOnAdd = (newMovie: Movie) => {
+    onAdd(newMovie);
+    setQuery('');
+    setMovie(null);
+    setIsLoading(false);
+  };
+
+  const handleSubmit = async (event: { preventDefault: () => void; }) => {
+    event.preventDefault();
+    setIsLoading(true);
+
+    const response = await getMovie(query);
+
+    setIsLoading(false);
+
+    if ('Error' in response) {
+      setErrorMessage(true);
+    } else {
+      const movieFromServer: Movie = {
```

Comment:
why create a variable and not jut pass an object into a function? you do not use it enywhere else